packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -472,7 +472,8 @@ async function getCompassExecutionParameters(): Promise<{
   );
   const binary = testPackagedApp
     ? getCompassBinPath(await getCompassBuildMetadata())
-    : require('electron');
+    : // eslint-disable-next-line @typescript-eslint/no-var-requires
+      (require('electron') as unknown as string);
```

Comment:
Really unsure why this started popping up as an issue for me tbh – feels like it should always have been complaining about this